pkg/Cpanel/Security/Advisor/Assessors/Brute.pm

```diff
@@ -28,6 +28,7 @@ package Cpanel::Security::Advisor::Assessors::Brute;
 
 use strict;
 use Cpanel::Config::Hulk ();
+use Cpanel::PsParser;
```

Comment:
Minor: We generally prefer `use Cpanel::PsParser ();` for modules, as that explicitly avoids importing functions.  Some simple imports are exempted from this, especially `Try::Tiny` & `Cpanel::Imports`.